### ISSUE TITLE
Nhentai didn't respond

### ISSUE THREAD

**Issue by 00dark** [Dar9586/NClientV2#752]
When I open Apk with internet, Apk automatically exits and shows no response but when internet is off then apk respond and I can see the downloaded gallery . 
What can I do now? Please fix this issue.

**Comment by HaremKnight** [comment]
Same here. I think Nhentai is trying to update their site and it breaks the client way of reading. Either the site reverse the update or the client update.

**Comment by TestUser1713** [comment]
You can still use the app if you turn off your wifi before opening the app, filter on popular and turn wifi back on. Just dont use anything other than Popular or it will crash again, and you have to repeat the wifi thing.

**Comment by HaremKnight** [comment]
> You can still use the app if you turn off your wifi before opening the app, filter on popular and turn wifi back on. Just dont use anything other than Popular or it will crash again, and you have to repeat the wifi thing.

Additionally, when I search any artist or group, it works too.

**Comment by 974115250** [comment]
Some Issue, It may be that the UserAgent setting Or Automatic update Check Cause This Problem

**Comment by HaremKnight** [comment]
> Some Issue, It may be that the UserAgent setting Or Automatic update Check Cause This Problem

No, it can still connect and function well when searching 'popular' or artist/group name so its probably not those two.

**Comment by uwuclxdy** [comment]
same 😔

**Comment by Soviena** [comment]
I think the problem is, the app cant display .webp image extension. If you download the most recent post from the app. Image extension is .webp not jpg.

**Comment by HaremKnight** [comment]
> I think the problem is, the app cant display .webp image extension. If you download the most recent post from the app. Image extension is .webp not jpg.

This is very likely. I tested to download some thumbnail on 'popular' and it does stay jpg or png. I tried to search some artist I tried before and find some untranslated doujin using webp format. If I filter to english, it doesn't crash.

**Comment by Kronos2308** [comment]
i buil the pull request and the crash not happend any more seems like some images not load
[UnsignedBuild](https://mega.nz/file/Q6p0yJZL#B31QF9MP-yLx6PjQwfIctfDZKFFjGe1Yvkm7dIrMCdU) 
this is a unsigned build 

> You can still use the app if you turn off your wifi before opening the app, filter on popular and turn wifi back on. Just dont use anything other than Popular or it will crash again, and you have to repeat the wifi thing.

![344105789_558236336455864_8351932237771625941_n](https://github.com/user-attachments/assets/359641e4-81f4-43ad-8395-21f3ba150414)

**Comment by HaremKnight** [comment]
> i buil the pull request and the crash not happend any more seems like some images not load [UnsignedBuild](https://mega.nz/file/Q6p0yJZL#B31QF9MP-yLx6PjQwfIctfDZKFFjGe1Yvkm7dIrMCdU) this is a unsigned build
> 
> > You can still use the app if you turn off your wifi before opening the app, filter on popular and turn wifi back on. Just dont use anything other than Popular or it will crash again, and you have to repeat the wifi thing.

Nice! When will these get released?

**Comment by Kronos2308** [comment]
seems like the probles is with the webp images

**Comment by Kronos2308** [comment]
[app-debug.zip](https://github.com/user-attachments/files/17713899/app-debug.zip)
need some test but seems like work

![imagen](https://github.com/user-attachments/assets/a1f07515-d944-467e-a4fd-2da58c379228)

**Comment by Kronos2308** [comment]
> > i buil the pull request and the crash not happend any more seems like some images not load [UnsignedBuild](https://mega.nz/file/Q6p0yJZL#B31QF9MP-yLx6PjQwfIctfDZKFFjGe1Yvkm7dIrMCdU) this is a unsigned build
> > > You can still use the app if you turn off your wifi before opening the app, filter on popular and turn wifi back on. Just dont use anything other than Popular or it will crash again, and you have to repeat the wifi thing.
> 
> Nice! When will these get released?

try #762

**Comment by 974115250** [comment]
> > i buil the pull request and the crash not happend any more seems like some images not load [UnsignedBuild](https://mega.nz/file/Q6p0yJZL#B31QF9MP-yLx6PjQwfIctfDZKFFjGe1Yvkm7dIrMCdU) this is a unsigned build
> > > You can still use the app if you turn off your wifi before opening the app, filter on popular and turn wifi back on. Just dont use anything other than Popular or it will crash again, and you have to repeat the wifi thing.
> 
> Nice! When will these get released?



> > i buil the pull request and the crash not happend any more seems like some images not load [UnsignedBuild](https://mega.nz/file/Q6p0yJZL#B31QF9MP-yLx6PjQwfIctfDZKFFjGe1Yvkm7dIrMCdU) this is a unsigned build
> > > You can still use the app if you turn off your wifi before opening the app, filter on popular and turn wifi back on. Just dont use anything other than Popular or it will crash again, and you have to repeat the wifi thing.
> 
> Nice! When will these get released?



> > i buil the pull request and the crash not happend any more seems like some images not load [UnsignedBuild](https://mega.nz/file/Q6p0yJZL#B31QF9MP-yLx6PjQwfIctfDZKFFjGe1Yvkm7dIrMCdU) this is a unsigned build
> > > You can still use the app if you turn off your wifi before opening the app, filter on popular and turn wifi back on. Just dont use anything other than Popular or it will crash again, and you have to repeat the wifi thing.
> 
> Nice! When will these get released?

https://github.com/user-attachments/files/17713899/app-debug.zip

It could work

**Comment by 974115250** [comment]
> [app-debug.zip](https://github.com/user-attachments/files/17713899/app-debug.zip) need some test but seems like work
> 
> ![imagen](https://private-user-images.githubusercontent.com/36446521/385217683-a1f07515-d944-467e-a4fd-2da58c379228.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE1MTI5MjYsIm5iZiI6MTczMTUxMjYyNiwicGF0aCI6Ii8zNjQ0NjUyMS8zODUyMTc2ODMtYTFmMDc1MTUtZDk0NC00NjdlLWE0ZmQtMmRhNThjMzc5MjI4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTEzVDE1NDM0NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWJjMWMyNjkyMzI3YmIwNWVmN2IyMmJkZDg5MThjYzRmMjZkZGVjNWI2NTY5NzdhNjRkOWE0MWU4NGJkMDU1N2ImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.C2kPYXq5ntB_Vbn4PJpSxIVa-xP71pBGw3oqXbqmoVw)

Thanks
